lib/home/library/video_library.dart

```diff
@@ -167,6 +167,29 @@ class _VideoLibraryState extends State<VideoLibrary> {
       return const Center(child: CircularProgressIndicator());
     }
 
+    final Widget blankCard = Card(
+      clipBehavior: Clip.antiAlias,
+      margin: const EdgeInsets.all(10),
+      color: Colors.white,
+      child: ListTile(
+        title: Container(
+          width: 100,
+          height: 20,
+          color: Colors.grey[300],
+        ),
+        subtitle: Container(
+          width: 150,
+          height: 20,
+          color: Colors.grey[200],
+        ),
+        leading: Container(
+          width: 100,
+          height: 100,
+          color: Colors.grey[400],
+        ),
+      ),
+    );
```

Comment:
We could probably make the `blankCard` a constant for better performance:

```suggestion
    const Widget blankCard = Card(
      clipBehavior: Clip.antiAlias,
      margin: EdgeInsets.all(10),
      color: Colors.white,
      child: ListTile(
        title: ColoredBox(
          color: Color(0xffbbbbbb),
          child: SizedBox(width: 100, height: 20),
        ),
        subtitle: ColoredBox(
          color: Color(0xffeeeeee),
          child: SizedBox(width: 150, height: 20),
        ),
        leading: ColoredBox(
          color: Color(0xffbbbbbb),
          child: SizedBox(width: 100, height: 100),
        ),
      ),
    );
```